snippets/java/LongestCommonSubsequence.md

```diff
@@ -0,0 +1,91 @@
+# Longest Common Subsequence
+
+## Code
+```Java
+import java.util.*;
+public class LCS3 {
```

Comment:
Provide better class names and better comments with explanation for a clearer understanding for others too.